.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "automl",
+  "name_pretty": "Cloud AutoML",
+  "product_documentation": "https://cloud.google.com/automl/docs/",
+  "client_documentation": "https://cloud.google.com/nodejs/docs/reference/automl/latest/",
+  "issue_tracker": "",
```

Comment:
there does not appear to be an issue tracker for the top-level `automl` library, there are issue trackers for sub-apis like vision.

Reply:
In that case, I'd send them to the AI platform issue tracker:
https://issuetracker.google.com/savedsearches/559744